--- src/hhd/__main__.py
+++ src/hhd/__main__.py
@@ -411,18 +411,6 @@
                     CONFIG_DIR,
                     join(CONFIG_DIR, "profiles"),
                 ]
-                for fn in cfg_fns:
-                    fd = os.open(expanduser(fn, ctx), os.O_RDONLY)
-                    fcntl.fcntl(
-                        fd,
-                        fcntl.F_NOTIFY,
-                        fcntl.DN_CREATE
-                        | fcntl.DN_DELETE
-                        | fcntl.DN_MODIFY
-                        | fcntl.DN_RENAME
-                        | fcntl.DN_MULTISHOT,
-                    )
-                    cfg_fds.append(fd)
 
                 should_initialize.clear()
                 logger.info(f"Initialization Complete!")
